sdk/cognitivelanguage/Azure.AI.Language.Text/samples/Sample5_AnalyzeTextAsync_RecognizePiiEntities.md

```diff
@@ -0,0 +1,87 @@
+# Recognizing Personally Identifiable Information in Documents
+
+This sample demonstrates how to recognize Personally Identifiable Information (PII) in one or more documents.
+
+## Create a `TextAnalysisClient`
+
+To create a new `TextAnalysisClient`, you will need the service endpoint and credentials of your Language resource. To authenticate, you can use the [`DefaultAzureCredential`][DefaultAzureCredential], which combines credentials commonly used to authenticate when deployed on Azure, with credentials used to authenticate in a development environment. In this sample, however, you will use an `AzureKeyCredential`, which you can create with an API key.
+
+```C# Snippet:CreateTextClient
+Uri endpoint = new Uri("https://myaccount.cognitiveservices.azure.com");
+AzureKeyCredential credential = new("your apikey");
+TextAnalysisClient client = new TextAnalysisClient(endpoint, credential);
+```
+
+The values of the `endpoint` and `apiKey` variables can be retrieved from environment variables, configuration settings, or any other secure approach that works for your application.
+
+## Recognizing Personally Identifiable Information in multiple documents
+
+To recognize Personally Identifiable Information in multiple documents, call `AnalyzeText` on an `TextPiiEntitiesRecognitionInput`.  The results are returned as a `AnalyzeTextPiiResult`.
+
+```C# Snippet:Sample5_AnalyzeTextAsync_RecognizePii
+string documentA =
+    "Parker Doe has repaid all of their loans as of 2020-04-25. Their SSN is 859-98-0987. To contact them,"
+    + " use their phone number 800-102-1100. They are originally from Brazil and have document ID number"
+    + " 998.214.865-68.";
+
+string documentB =
+    "Yesterday, Dan Doe was asking where they could find the ABA number. I explained that it is the first"
+    + " 9 digits in the lower left hand corner of their personal check. After looking at their account"
+    + " they confirmed the number was 111000025.";
+
+string documentC = string.Empty;
+
+AnalyzeTextInput body = new TextPiiEntitiesRecognitionInput()
+{
+    TextInput = new MultiLanguageTextInput()
+    {
+        Documents =
+        {
+            new MultiLanguageInput("A", documentA) { Language = "en" },
+            new MultiLanguageInput("B", documentB) { Language = "es" },
+            new MultiLanguageInput("C", documentC),
+        }
+    },
+    ActionContent = new PiiActionContent()
+    {
+        ModelVersion = "latest",
+    }
+};
+
+Response<AnalyzeTextResult> response = await client.AnalyzeTextAsync(body);
+AnalyzeTextPiiResult piiTaskResult = (AnalyzeTextPiiResult)response.Value;
+
+foreach (PiiResultWithDetectedLanguage piiResult in piiTaskResult.Results.Documents)
+{
+    Console.WriteLine($"Result for document with Id = \"{piiResult.Id}\":");
+
```

Comment:
Do we also want to highlight redactedText in the document here?

Reply:
Added